invenio_access/translations/messages.pot

```diff
@@ -1,22 +1,22 @@
 # Translations template for invenio-access.
-# Copyright (C) 2022 CERN
+# Copyright (C) 2023 CERN
```

Comment:
In some cases this gets changed when running tests, which is unfortunate. Could you uncommit this change?